frontend/src/service/LessonService.ts

```diff
@@ -25,4 +25,9 @@ export default class LessonService {
     saveNewMessage(lessonId : string ,message:Message) {
         return axios.post(`/api/lessons/message/${lessonId}`,message);
     }
+    deleteMessage(lessonId : string ,message:Message) {
+        return axios.delete(`/api/lessons/delete-message/${lessonId}`, {
+            data: message
+        });
+    }
```

Comment:
This should work with
`return axios.delete("/api/lessons/delete-message/${lessonId}", message);`
since your frontend message type and your backend message model are quasi-identical.

Reply:
I get an error when I do that :(